"shm:" string is missing on I_MPI_FABRICS option.

TeppeiIshii-zz opened the pull request:
"shm:" string is missing on I_MPI_FABRICS option.

The correct one should be:
mpirun -hosts <host1>,<host2> -ppn 1 -n 2 -env I_MPI_FABRICS=shm:dapl -env I_MPI_DAPL_PROVIDER=ofa-v2-ib0 -env I_MPI_DYNAMIC_CONNECTION=0 IMB-MPI1 pingpong

This is customer reported issue caused tons of compute money as the customer kept failing with HPC VM's.

azurecla commented:
Hi __@chrisckchang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Teppei-MSFT committed:
"shm:" string is missing on I_MPI_FABRICS option.

"shm:" string is missing on I_MPI_FABRICS option.

The correct one should be:
mpirun -hosts <host1>,<host2> -ppn 1 -n 2 -env I_MPI_FABRICS=shm:dapl -env I_MPI_DAPL_PROVIDER=ofa-v2-ib0 -env I_MPI_DYNAMIC_CONNECTION=0 IMB-MPI1 pingpong

This is customer reported issue caused tons of compute money as the customer kept failing with HPC VM's.
